app/models/payment_gateway.rb

```diff
@@ -56,8 +56,9 @@ def self.find(type)
   #     BraintreeOrangeGateway if :login is given or BraintreeeBlueGateway otherwise
   #
   #
-  def self.implementation(type)
-    ActiveMerchant::Billing::Base.gateway(type)
+  def self.implementation(type, **options)
+    specific_type = (options[:sca] && type == :stripe) ? :stripe_payment_intents : type
```

Comment:
Hmm. I'd rather see this detail abstracted inside the `ActiveMerchant::Billing::StripeGateway` class instead of defining a new gateway type.

Reply:
I need a call with you about this as soon as you can 😄

Reply:
oh! I get it now. My bad. Sorry. It's ActiveMerchant class, not ours. Please ignore.